GPU/GPUTracking/Merger/GPUTPCGMO2Output.cxx

```diff
@@ -57,6 +57,9 @@ GPUdii() void GPUTPCGMO2Output::Thread<GPUTPCGMO2Output::prepare>(int nBlocks, i
     if (nCl == 0) {
       continue;
     }
+    if (merger.Param().rec.dropSecondaryLegsInOutput && nCl + 2 < GPUCA_TRACKLET_SELECTOR_MIN_HITS(tracks[i].GetParam().GetQPt())) { // Give 2 hits tolerance in the primary leg, compared to the full fit of the looper
```

Comment:
Thanks. I think ``GPUCA_TRACKLET_SELECTOR_MIN_HITS`` should depend not on q/pt, but on the curvature. Perhaps, to avoid extra multiplications,  instead of thresholds T=10 and 5. one should simply use precalculated thresholds  ``abs(B)>1e-3 ? T*5./B : T`` ?

Reply:
You are right, that should be done in multiple places. But also this is a first fix for your problem already
Actually I created the PR mainly to trigger a full CI build to test if the new full-system-test CI works.